### DISCUSSION TITLE
antd-mobile@5 初始化脚手架

### DISCUSSION THREAD

**Discussion by xiaohuoni** [ant-design/ant-design-mobile#4699] [Show and tell]
https://github.com/ant-design/antd-mobile-template

<img src="https://user-images.githubusercontent.com/11746742/175474539-d7c06762-e10c-4597-8fc2-ec2ef6e9b2ae.png" width="200px"/>
<img src="https://user-images.githubusercontent.com/11746742/175474586-df2400ff-c4f1-43c5-8575-101967d53927.png" width="200px"/>

- [x] Umi@4 方案，默认 mfsu
- [x] 支持移动端屏幕适配
- [x] 添加 keepalive
- [x] 固定头部的 list 页面
- [x] 全局的 tabbar 布局

最近想尝鲜 antd-mobile@5 的同学可以使用 [use this template](https://github.com/ant-design/antd-mobile-template/generate) 来快速创建项目。
<a href="https://github.com/ant-design/antd-mobile-template/generate"><img src="https://user-images.githubusercontent.com/11746742/133240587-04118b18-506a-4f5b-83bd-ac45d7a2fd7c.png" /></a>

**Comment by konglx90** [comment]
啥时候发正式版呀？

**Comment by awmleer** [comment]
> 啥时候发正式版呀？

#3924

**Comment by bianliuzhu** [comment]
使用CRA创建的react项目,安装5.0的版本, 安装成功,使用时会报,对应模块找不到: `Module not found: Can't resolve 'antd-mobile/lib/toast'` 有没有比较好的解决办法

**Comment by lwQin** [comment]
手动更新 `antd-mobile` 包后缩放比例变了
<img width="462" alt="Xnip2021-11-12_16-59-02" src="https://user-images.githubusercontent.com/11586662/141439854-f51f958d-08e8-44d4-9601-401d54a0c870.png">
<img width="491" alt="Xnip2021-11-12_16-59-10" src="https://user-images.githubusercontent.com/11586662/141439869-b59c6a93-804b-4c61-a258-470531a6a2c9.png">

**Comment by awmleer** [comment]
@xiaohuoni 这脚手架可以更新一下了……

**Comment by awmleer** [comment]
用 @umijs/plugin-antd-mobile

**Comment by xiaohuoni** [comment]
@lwQin @awmleer 完成 https://github.com/ant-design/antd-mobile5-template/commit/dba40e12256b59c7d7cd8b5be6b2f23f896352ff

**Comment by xiaohuoni** [comment]
模版已更新到 umi@4 方案
